app/views/state_file/questions/md_permanently_disabled/edit.html.erb

```diff
@@ -38,20 +38,10 @@
 
       <div class="question-with-follow-up__follow-up" id="primary-disability-proof">
         <div class="white-group">
-          <div class="question-with-follow-up">
-            <div class="question-with-follow-up__question">
-                <% show_disability_warning =  !current_intake.filing_status_mfj? ? "#disability-warning" : nil %>
-                <%= f.cfa_radio_set(:primary_proof_of_disability_submitted, label_text: t(".primary_proof_question_html"), collection: [
-                  { value: "yes", label: t("general.affirmative") },
-                  { value: "no", label: t("general.negative"), input_html: { "data-follow-up": show_disability_warning }},
-                ]) %>
-              </div>
-            <div class="question-with-follow-up__follow-up" id="disability-warning">
-                <div class="notice">
-                  <p><%= t(".unfortunately_dont_support_html", path: questions_md_pension_exclusion_offboarding_path) %></p>
-                </div>
-              </div>
-          </div>
```

Comment:
Was it just to much of a pain to have the notice buried so deep in question with follow ups?

Reply:
No, it's that clicking off the page here would cause this page's form to never get submitted. There were various shenanigans proposed about having this link also submit the form, or having the Continue button on the softboarding page somehow remember and set the fields to what the user had selected on this page, but I spoke with design and we felt that just having all users consistently see softboarding as the next page in the flow, if it was applicable, was reasonable from a user perspective and easier to implement.

Reply:
Cool thanks!